content/join/policy.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Policy
+parent: Joining the infrastructure
+nav_order: 10
+---
+
+# Policy
+
+In order to ensure proper testbed operation and maintenance we require that all sites (external and internal) abide by the policies outlined below.
+
+1. Provide one or more dedicated machines on-site to act as the SCIONLab nodes. The machine should not be used for any other tasks beyond its role as an SCIONLab node.
+2. Provide root (or sudo) access to the SCIONLab management institution (currently ETH Zurich).
+3. Allow the SCIONLab management institution to install, remove and update SCIONLab related software as needed for the operation and maintenance of the network.
+4. Work with the SCIONLab management institution to solve any firewall and tunneling issues that may arise during the installation, configuration and operation of the SCIONLab node(s).
+5. Provide the name, email and telephone information of an easily and quickly reachable site-specific operator who will be the main contact whenever issues arise with the local SCIONLab node(s). A backup operator, while not required, is highly encouraged. Operator email addresses must belong to the institution. All local operators will be required to join and regularly monitor an SCIONLab community mailing list and are required to respond to requests from the testbed management institution in a reasonable time, but no more than 3 working days.
+6. Ensure that the machine has limited logins beyond the SCIONLab account. A local operator account is highly encouraged, but general user accounts should not be present on the SCIONLab node. In addition, only SCIONLab-related software should be allowed to run on the machine.
+7. The machine may be a dedicated physical box or a virtual machine with a publicly routable IP address (i.e., should not be behind a NAT). The local operators should ensure that if the machine has usage limits or other restrictions (for example VMs on cloud services), those will not interfere with proper testbed operation.
+8. In case more than one machine is being provided by the institution, it is allowed if only one of them has a publicly routable IP address whereas the others use private static IP.
```

Comment:
Why?

Reply:
This is to cover the following use case
* multi-host AS
* border router has a public static IP for the uplink connectivity
* services inside AS talk with each other using static private IPs (because public are not needed as long as inside-AS connectivity works)

Reply:
what about having having multiple border routers for uplink?

Reply:
This is a valid scenario, we should then write something closer to "machines hosting border routers must have a public static IPv4 address"